Packs/ExpanseV2/ReleaseNotes/1_7_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Expanse v2
+- Added support for the **get-modified-remote-data** command which returns a list of modified incidents since the last update.
```

Comment:
```suggestion
- Added support for the **get-modified-remote-data** command which returns a list of modified incidents since the last update (used in the incident mirroring process from XSOAR version 6.1.0).
```